spec/std/syscall_spec.cr

```diff
@@ -1,4 +1,4 @@
-{% skip_file unless flag?(:linux) %}
+{% skip_file unless flag?(:linux) && !flag?(:interpreted) %}
```

Comment:
**question:** Should we mark this as pending for the interpreter? I see no reason why we shouldn't eventually get syscall supported.

Reply:
Windows doesn't have `pending` either, so I believe this is fine?